sdks/python/setup.py

```diff
@@ -108,6 +108,7 @@ def get_version():
     # TODO(BEAM-2964): Remove the upper bound.
     'six>=1.9,<1.11',
     'typing>=3.6.0,<3.7.0',
+    'futures>=2.2.0',
```

Comment:
Why are we not using the latest version?
Could you include an upper bound also (e.g `,<4`)

Reply:
I imposed the min version requirement to be same as grpcio which was "futures [required: >=2.2.0, installed: 3.1.1]".
But having latest version makes sense.
Sure I can put the upperbound to <4